runtime-display/vulkan-loader/spec

```diff
@@ -1,4 +1,4 @@
-VER=1.3.236
+VER=1.3.269
 SRCS="tbl::https://github.com/KhronosGroup/Vulkan-Loader/archive/v$VER.tar.gz"
```

Comment:
Please try not to use automatically packaged tarballs, use Git tags instead.

Reply:
Fixed.